find_attitude/make_distances.py

```diff
@@ -14,7 +14,7 @@
 MAX_ACA_DIST = 2.0 * u.deg  # degrees corner to corner (actual max is 7148 arcsec)
 MAX_MAG = 10.0  # Max mag to include in the distances file
 DATE_DISTANCES = "2025:001"  # Date for proper motion correction in distances
-HEALPIX_NSIDE = 16
+HEALPIX_NSIDE = 64
```

Comment:
It seems like it should be getting this from the agasc module code but ...

Reply:
This NSIDE is independent of the `agasc` module code. It is set by the NSIDE which ends up giving tiles that are around one square degree, which is a convenient number here. In particular the index fits in 16 bits and the size matches the scale of the ACA FOV.